tests/testthat/_snaps/project.md

```diff
@@ -1,8 +1,23 @@
-# renv_project_synchronized_check() reports if no packages installed
+# renv_project_synchronized_check() installs packages if needed
 
     Code
       ok <- renv_project_synchronized_check()
     Output
       * None of the packages recorded in the lockfile are installed.
-      * Use `renv::restore()` to restore the project library.
+      * Using `renv::restore()` to restore the project library.
+      The following package(s) will be updated:
+      
+      # CRAN ---
+      - bread       [* -> 1.0.0]
+      - breakfast   [* -> 1.0.0]
+      - oatmeal     [* -> 1.0.0]
+      - toast       [* -> 1.0.0]
+      
+      # Downloading packages ---
+      [no downloads required]
+      
+      Installing bread     ... OK [linked from cache]
```

Comment:
No "# Installing packages ---" header before here?

Reply:
Ooops, I guess we never added that to restore